popup/popups.js

```diff
@@ -0,0 +1,211 @@
+const projects = [
+  {
+    id: 'popup1',
+    Title: 'Tonic',
+    closeButt: '&times;',
+    Client: 'CANOPY',
+    counter1: 'images/Dot-shape.png',
+    Role: 'Back End Dev',
+    counter2: 'images/Dot-shape.png',
+    year: '2015',
+    snapshot: 'images/Snap1.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic. typesetting, remaining essent.",
+
+    badges: ['html', 'css', 'javascript'],
+    liveBtn: 'See live',
+    sourceBtn: 'See Source',
```

Comment:
[OPTIONAL] 👍🏽 
- Kindly ensure that your `See Live` and `See Sourece` buttons are active links. You can connect them to their individual links using `href`. 🤗

Reply:
@Favourezeugwa Thank you